src/EditorFeatures/Core/ValueTracking/ValueTrackedItem.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.CodeAnalysis.ValueTracking
+{
+    internal class ValueTrackedItem
```

Comment:
Could this be a record?

Reply:
Sure, why not :)

Reply:
Actually, can we use records in Roslyn yet? 

```
Severity	Code	Description	Project	File	Line	Suppression State
Error	CS0518	Predefined type 'System.Runtime.CompilerServices.IsExternalInit' is not defined or imported	Microsoft.CodeAnalysis.EditorFeatures (netcoreapp3.1)
```